dynamic_preferences/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "1.7"
+__version__ = "1.7.1"
```

Comment:
```suggestion
__version__ = "1.7"
```

No need to update this, I'll do it during the release :)